system-tests/projects/angular-custom-root/ui/main.ts

```diff
@@ -0,0 +1,8 @@
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic'
```

Comment:
I wonder if we need a basic Angular project in `project-fixtures` at some point, so we iterate over it like we do for React in system tests, testing against each supported major :thinking:

Reply:
We do have one, but this project felt different enough that I thought I'd make a new one. If I was to reuse the `angular` project-fixture, I'd have had to overwrite almost every file (every config file + the src code is in a different directory)

Reply:
I agree with @ZachJW34  that this is worth a different project since the source root is pretty fundamental part of the app.